gradle-plugin/src/main/kotlin/co/touchlab/kotlin/gradle/targets/native/tasks/CocoapodsTasks.kt

```diff
@@ -32,14 +32,14 @@ open class PodspecTask : DefaultTask() {
     @OutputFile
     val outputFile: File = project.projectDir.resolve("$specName.podspec")
 
-    @Input
-    val frameworkNameProvider: Provider<String> = project.provider { framework.baseName}
+    @get:Input
+    lateinit var frameworkName:String
 
-    @get:Nested
-    internal lateinit var settings: CocoapodsExtension
+    @get:Input
+    var isStatic:Boolean = true
```

Comment:
It looks like you shouldn't set these directly, but they're public. I don't want to hold up this PR, but at some point I'd like to think about that.